### ISSUE TITLE
Error parsing FLEX API error response

### ISSUE THREAD

**Issue by davidburulic** [CyberSource/cybersource-flex-samples-java#29]
Hello,

when I tried to run your jsp-microform example, I've encountered the following exception:

org.apache.jasper.JasperException: An exception occurred processing [/index.jsp] at line [80]
...
javax.servlet.ServletException: com.cybersource.flex.sdk.exception.FlexSDKInternalException: Error parsing FLEX API error response: [responseStatus is expected]
...

I am using: 
Java 1.8.0_112
Tomcat 8.5.47
Maven 3.3.9 (pom.xml specifies version 3.2.0)
JS Flex Microform lib 0.4.0
flex-server-sdk 0.3.0

I've followed the instructions in README.md for build and setup of the example.

I kindly ask you for further advice.

**Comment by manoharan0308** [comment]
Check your actual response from the HttpClient. I had the same issue, but when I debugged it turn out to be an authentication error(401). During the non successful cases, the HttpClient doesn't return the "responseStatus".

**Comment by vshinde83** [comment]
Any updates?

**Comment by davidburulic** [comment]
We managed to make Flex microform work as expected.

What I would advise to anyone, who runs into a similar issue, is to double check all of the fields which are posted to CyberSource services. Fields should be properly signed, declared, values must be valid etc... Alo merchant keys and credentials must be properly setup, otherwise errors can be expected, as @manoharan0308 mentioned.
I think it's worth mentioning to look at how merchant profile is configured on CyberSource business backend, so that everything is properly configured in order for Flex microform to work (which fields are required etc...).

I hope this helps anyone who's struggling with this integration.

I'm closing this thread since we were able to resolve it.
